docs/reference/cpp/tools.md

```diff
@@ -0,0 +1,115 @@
+# Tools
+
+A lot goes into making a well structured C++ project, much more than any one team should have to do.
+
```

Comment:
Do you use debugging tools like GDB? It might be worth mentioning them here if you use them often.

Reply:
Good point, I'll likely need to elaborate on the specific quirks of GDB with VSCode.